[2025-02-18T19:05:14.492+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-18T19:05:14.508+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: example_bash_operator.run_after_loop manual__2025-02-18T19:04:55+00:00 [queued]>
[2025-02-18T19:05:14.516+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: example_bash_operator.run_after_loop manual__2025-02-18T19:04:55+00:00 [queued]>
[2025-02-18T19:05:14.516+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-02-18T19:05:14.639+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): run_after_loop> on 2025-02-18 19:04:55+00:00
[2025-02-18T19:05:14.644+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=336) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-18T19:05:14.645+0000] {standard_task_runner.py:72} INFO - Started process 340 to run task
[2025-02-18T19:05:14.646+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'example_bash_operator', 'run_after_loop', 'manual__2025-02-18T19:04:55+00:00', '--job-id', '13', '--raw', '--subdir', '/home/***/.local/lib/python3.12/site-packages/***/example_dags/example_bash_operator.py', '--cfg-path', '/tmp/tmpxnq0u0y8']
[2025-02-18T19:05:14.647+0000] {standard_task_runner.py:105} INFO - Job 13: Subtask run_after_loop
[2025-02-18T19:05:14.683+0000] {task_command.py:467} INFO - Running <TaskInstance: example_bash_operator.run_after_loop manual__2025-02-18T19:04:55+00:00 [running]> on host cd6c965fea54
[2025-02-18T19:05:14.736+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='example_bash_operator' AIRFLOW_CTX_TASK_ID='run_after_loop' AIRFLOW_CTX_EXECUTION_DATE='2025-02-18T19:04:55+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-18T19:04:55+00:00'
[2025-02-18T19:05:14.738+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-02-18T19:05:14.738+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-02-18T19:05:14.739+0000] {logging_mixin.py:190} INFO - Current task name:run_after_loop state:running start_date:2025-02-18 19:05:14.508956+00:00
[2025-02-18T19:05:14.739+0000] {logging_mixin.py:190} INFO - Dag name:example_bash_operator and current dag run status:running
[2025-02-18T19:05:14.739+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-18T19:05:14.740+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-02-18T19:05:14.741+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo https://***.apache.org/']
[2025-02-18T19:05:14.753+0000] {subprocess.py:99} INFO - Output:
[2025-02-18T19:05:14.760+0000] {subprocess.py:106} INFO - https://***.apache.org/
[2025-02-18T19:05:14.761+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-02-18T19:05:14.786+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-18T19:05:14.786+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=example_bash_operator, task_id=run_after_loop, run_id=manual__2025-02-18T19:04:55+00:00, execution_date=20250218T190455, start_date=20250218T190514, end_date=20250218T190514
[2025-02-18T19:05:14.810+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-02-18T19:05:14.811+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-02-18T19:05:14.811+0000] {logging_mixin.py:190} INFO - Dag name:example_bash_operator queued_at:2025-02-18 19:05:02.270377+00:00
[2025-02-18T19:05:14.812+0000] {logging_mixin.py:190} INFO - Task hostname:cd6c965fea54 operator:BashOperator
[2025-02-18T19:05:14.827+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-18T19:05:14.855+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-18T19:05:14.857+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
